Library/Formula/cv.rb

```diff
@@ -0,0 +1,15 @@
+class Cv < Formula
+  desc "Coreutils Viewer: Show progress for coreutils"
+  homepage "https://github.com/BestPig/cv"
```

Comment:
This is still pointing to the fork... see #32161.

Shouldn't this point to the official repo instead? https://github.com/Xfennec/cv